apps/web/app/environments/[environmentId]/surveys/[surveyId]/responses/SingleResponse.tsx

```diff
@@ -73,6 +76,19 @@ export default function SingleResponse({ data, environmentId, surveyId }: OpenTe
     setIsDeleting(false);
   };
 
+  const tooltipContent = (
+    <TooltipContent>
```

Comment:
please only show tooltip if personAttributes is not empty to avoid a tooltip without content